clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.h

```diff
@@ -17,6 +17,19 @@
 #include "mlir/Transforms/DialectConversion.h"
 #include "clang/CIR/Dialect/IR/CIRDialect.h"
 
+namespace cir {
+class SqrtOp;
+}
+
+class CIRToLLVMSqrtOpLowering : public mlir::OpConversionPattern<cir::SqrtOp> {
```

Comment:
You shouldn't need this. In the upstream CIR implementation, these definitions are automatically generated by TableGen.